### ISSUE TITLE
LC0003 Do not use an Object ID for properties - False positive on array[2] of Record ...

### ISSUE THREAD

**Issue by rvanbekkum** [StefanMaron/BusinessCentral.LinterCop#56]
Hi @StefanMaron,
It seems like rule LC0003 gives a false positive on array variables. Here is an example:
```al
var
    CustCalendarChange: array[2] of Record "Customized Calendar Change";
```
![image](https://user-images.githubusercontent.com/7383241/135854692-e721c0f0-86b9-4715-9b05-568c6f1c6e2c.png)

**Comment by rvanbekkum** [comment]
Unfortunately, both warnings (false positives for LC0003 and LC0005) still persist on the following example: 🙁
```al
var
    CustCalendarChange: array[42] of Record "Customized Calendar Change" temporary;
```
